rdflib/plugins/stores/auditable.py

```diff
@@ -56,9 +56,10 @@ def add(self, triple, context, quoted=False):
             context = context.__class__(self.store, context.identifier) if context is not None else None
             ctxId = context.identifier if context is not None else None
             self.reverseOps.append((s, p, o, ctxId, 'remove'))
-            if (s, p, o, ctxId, 'add') in self.reverseOps:
-                self.reverseOps.remove(
-                    (s, p, o, context, 'add'))
+            try:
+                self.reverseOps.remove((s, p, o, ctxId, 'add'))
+            except ValueError:
+                pass
```

Comment:
not so much a comment for this change, but for my understanding: is this to "compress" the reverseOps only?